config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-presubmits.yaml

```diff
@@ -0,0 +1,48 @@
+presubmits:
+  kubernetes-sigs/azurefile-csi-driver:
+  - name: pull-azurefile-csi-driver-unit
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - test
+  - name: pull-azurefile-csi-driver-sanity
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - test-sanity
+  - name: pull-azurefile-csi-driver-integration
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - test-integration
+  - name: pull-azurefile-csi-driver-e2e
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - test-e2e

```

Comment:
@andyzhangx Do we need to configure Azure accounts for e2e tests?

Reply:
I think yes

Reply:
Will this https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/master/hack/run-e2e-test.sh be the right place for doing that ?

Reply:
maybe we can refer to https://github.com/kubernetes/test-infra/blob/2228dd155e83c96434e0b9ad45f6c570d526c1f7/config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml#L98 for e2e test

Reply:
pls use this config for reference: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-azure/sig-azure-config.yaml
Hi @ritazh, do you know about how to enable this azure e2e test on csi driver? The above config runs conformance test, while for csi driver, it should run PV related test.

Reply:
@andyzhangx Let's test the configure on our own prow first. Could you add me to the csi repo admins? I need to update hooks for setup the prow job

Reply:
@andyzhangx To run conformance test against a cluster with the azurefile csi driver, I would first create a cluster with your csi driver installed, then run conformance test against it. 

Take a look at this job: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml#L274-L325 It is running conformance test against a cluster that was created with aks-engine using a template that has `customCcmImage` configured https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml#L315 Although I'm not sure if the azurefile csi driver has been added to aks-engine yet.